addons/sale_project/models/sale_order_line.py

```diff
@@ -260,8 +260,11 @@ def _timesheet_create_task_prepare_values(self, project):
         if self.product_id.service_type not in ['milestones', 'manual']:
             allocated_hours = self._convert_qty_company_hours(self.company_id)
         sale_line_name_parts = self.name.split('\n')
-        title = sale_line_name_parts[0] or self.product_id.name
-        description = '<br/>'.join(sale_line_name_parts[1:])
+        title = sale_line_name_parts[0] if not self.order_id.sale_order_template_id else self.product_id.name
+        # If we have a quotation template we get the description of the template line related to the self sale
+        # order line. If not we fallback on the description in the order line itself.
+        description = (self.order_id.sale_order_template_id.sale_order_template_line_ids.filtered(lambda line: line.product_id == self.product_id).name
+                       or '<br/>'.join(sale_line_name_parts[1:]))
```

Comment:
Why not using the template as a fallback instead? 

```suggestion
        so_line_template_name = self.order_id.sale_order_template_id.sale_order_template_line_ids.filtered(lambda line: line.product_id == self.product_id).name or ''
        so_line_template_name_parts = so_line_template_name.split('\n')
        title = sale_line_name_parts[0] or so_line_template_name_parts[0] self.product_id.name
        description = '<br/>'.join(sale_line_name_parts[1:] or sale_line_template_name_parts[1:]))
```

What do you think? 

Maybe I miss something?

Reply:
@yoba-odoo ping 🙂

Reply:
@xavierbol oupsie :sweat_smile: Looking into it now :ok_hand:

Reply:
No :melting_face: 
https://github.com/odoo/odoo/blob/d3ab7b2c3ceddbb4cb8d8e91fac0d60e28e5673a/addons/sale_management/models/sale_order_template_line.py#L40-L43
The definition of the fields here in `sale.order.template.line` the 'Description' field variable called `name`

Reply:
Okay, but it does not mean, the field does not contain the name of the product. 🙂 

To me, this field will contain the both as we have in the `name` field in `sale.order.line`. 🤷🏼‍♂️ 

https://github.com/odoo/odoo/blob/master/addons/sale/models/sale_order_line.py#L119-L122

Reply:
Ah indeed, strange. :o